src/screens/CreateWalletScreen.tsx

```diff
@@ -112,7 +120,9 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
           email,
           flowType: ConfirmAddressFlowType.SUBSCRIBE,
           walletsToSubscribe: [wallet],
-          onSuccess: this.navigateToSuccesfullNotificationSubscriptionMessage,
+          onSuccess: isAfterAirdrop()
+            ? this.navigateToSuccesfullNotificationSubscriptionMessage
+            : () => this.navigateToAirdropWalletSubscription(wallet, true),
```

Comment:
seems that missing logic, redirecting to airdrop if the user clicks `no` on notifications

Reply:
not sure I understand but if user clicks NO on email notifications, it *should* still redirect to airdrop, only success message will be different at the end of wallet creation. 
What logic is missing?

Reply:
yes exactly, and I don't see this navigating to airdrop if the user clicks "no" on notifications

Reply:
done